gemhardware/include/gem/hw/optohybrid/HwOptoHybrid.h

```diff
@@ -1201,82 +516,76 @@ namespace gem {
            * @returns uint32_t 24 bit mask
-           * The mask has a 1 for VFATs that will not receive a broadcast request
-           * The mask has a 1 for VFATs whose tracking data will be ignored
+           *          The mask has a 1 for VFATs that will not receive a broadcast request
+           *          The mask has a 1 for VFATs whose tracking data will be ignored
            */
           uint32_t getConnectedVFATMask(bool update=false);
 
           /**
-           * Sends a write request for all setup registers on each VFAT specified by the mask
-           * @param uint8_t vthreshold1 value to write
-           * @param uint8_t vthreshold2 value to write
-           * @param uint8_t latency value to write
-           * @param uint32_t broadcastMask is the list of VFATs to send the broadcast commands to
+           * @brief Sends a write request for all setup registers on each connected VFAT
+           *        Uses the settings stored in the CTP7 BRAM
            */
-          void setVFATsToDefaults(uint8_t  const& vt1,
-                                  uint8_t  const& vt2,
-                                  uint8_t  const& latency,
-                                  uint32_t const& broadcastMask);
+          void setVFATsToDefaults();
+          void configureVFATs();
 
           /**
-           * Sends a write request for all setup registers on each VFAT specified by the mask
-           * @param std::map<std::string,uint8_t> map of VFAT register name to value to broadcast
-           * @param uint32_t broadcastMask is the list of VFATs to send the broadcast commands to
+           * @brief Sends a write request for all setup registers on each VFAT specified by the mask
+           * @param std::map<std::string, uint16_t> map of VFAT register name to value to broadcast
            */
-          void setVFATsToDefaults(std::map<std::string, uint8_t> const& regvals,
-                                  uint32_t const& broadcastMask);
-
+          void setVFATsToDefaults(std::map<std::string, uint16_t> const& regvals);
+          void configureVFATs(std::map<std::string, uint16_t> const& regvals);
 
-          const uhal::HwInterface& getOptoHybridHwInterface() const {
-            return dynamic_cast<const uhal::HwInterface&>(*this); };
+          /**
+           * @brief Sends a write request for all configuration registers on all GBTx chips
+           * @param gbtID is the GBTx to which the configuration will be written
+           * @param gbtcfg is a pointer to an array of 92 32-bit words
+           * @param gbtcfg is an array of 92 32-bit words
+           */
 
-          std::vector<linkStatus> getActiveLinks() { return v_activeLinks; }
-          bool isLinkActive(int i) { return b_links[i]; }
+          void configureGBT(uint8_t const& gbtID, uint32_t const* gbtcfg); // FIXME, make private
+          void configureGBT(uint8_t const& gbtID, std::array<const uint32_t, 92> const& gbtcfg);
```

Comment:
Either because I felt that the promise should be made that the values would not be changed, or because I was matching a declaration in the `ctp7_modules`, again, this type is expected to be `include`'d from a common `constants` definition